Makefile

```diff
@@ -11,7 +11,7 @@ DOCKER ?= docker
 DOCKER_BUF := $(DOCKER) run --rm -v $(CURDIR):/workspace --workdir /workspace bufbuild/buf
 
 GOFLAGS:=""
-GOTAGS:=
+GOTAGS:=ledger
```

Comment:
is this the only change needed to make ledger work with margin open/close?

Reply:
This is just to enable it for the command line binary for any transactions. The backend support for margin is already working.